app/liquid_tags/github_tag/github_issue_tag.rb

```diff
@@ -46,9 +53,29 @@ def parse_link(link)
     end
 
     def generate_api_link(input)
-      input = input.gsub(/\?.*/, "")
-      input = input.gsub(/\d{1,}#issuecomment-/, "comments/") if input.include?("#issuecomment-")
-      "https://api.github.com/repos/#{input.gsub(/.*github\.com\//, '')}"
+      uri = URI.parse(input).normalize
+      uri.host = nil if uri.host == "github.com"
+
+      # public PRs URLs are "/pull/{id}" but the API requires "/pulls/{id}"
```

Comment:
😂 I bet there was some good facepalming when they realized this discrepancy

Reply:
It's even weirder as the actual API call was done on the right URL thanks to a following string replace, it was just the URL stored in the DB that is invalid. But it works now because we generate the wrong URL which matches with the wrong URL stored in the DB :D